tests/test_tokenization_common.py

```diff
@@ -578,6 +579,25 @@ def test_tokenizers_common_ids_setters(self):
                 self.assertListEqual(getattr(tokenizer, "additional_special_tokens"), [token_to_test_setters])
                 self.assertListEqual(getattr(tokenizer, "additional_special_tokens_ids"), [token_id_to_test_setters])
 
+    @parameterized.expand([(True,), (False,)])
+    def test_tokenizers_common_properties_unset(self, verbose):
```

Comment:
A small suggestion to rename the test as I think it is specifically about "special tokens". WDYT? :blush: 

```suggestion
    def test_tokenizers_special_tokens_properties_unset(self, verbose):
```

Reply:
Sure, let's rename the test case.